### PR TITLE
fix: bring all comments `TODO` test back

### PR DESCRIPTION
**Pull request by yihong0618**
This `PR` fix all the todo tests
1. fix python3_error by add init file because that different  python versions have different format
2. fix uda_python3_test by drop the flaky test because it used `pickle` that are different for python3 versions
3. fix and make faultinject_python_test can run locally by use `Docker label` filter.

also add prepare scrpits for locally tests.

### CODE REVIEW

**Comment by Sasasu** [comment]
https://github.com/greenplum-db/plcontainer/blob/5f9871867a88a928fc6fdbdfa63074e8899ea73a/tests/regress/sql/faultinject_python.sql#L30-L32

I haven't tested this, but I think adding `docker ps -a --filter "label=dbid=2"` to faultinject_python.sql will fix the running count problem.

```
\! ssh `psql -d ${PL_TESTDB} -c 'select address from gp_segment_configuration where dbid=2' -t -A` docker ps -a --filter "label=dbid=2" </dev/null | wc -l
```

PL/C will add many labels to the docker process.

https://github.com/greenplum-db/plcontainer/blob/5f9871867a88a928fc6fdbdfa63074e8899ea73a/src/plc_docker_api.c#L307-L311

the old test required the host did not have any docker container running. I think we should fix this for the developer experience.

( I've never run a complete test locally.....

File: tests/regress/expected/uda_python3.out [hunk 1]
```diff
@@ -35,43 +35,3 @@ select sum_cols(i,j) FROM t;
 (1 row)
 
 drop table t;
--- Test GD(global dictionary)
-CREATE or replace FUNCTION pystack(state text, groupid text, vec text)
```
Comment (Sasasu) [inline tests/regress/expected/uda_python3.out:39]: `function_python3.sql` have a test case for `GP` access. remove this test case is fine.

**Review by Sasasu — APPROVED** [review]

File: tests/regress/expected/uda_python3.out (hunk 1 above)
Comment (yihong0618) [reply tests/regress/expected/uda_python3.out:39]: thanks.~

**Comment by yihong0618** [comment]
> https://github.com/greenplum-db/plcontainer/blob/5f9871867a88a928fc6fdbdfa63074e8899ea73a/tests/regress/sql/faultinject_python.sql#L30-L32
> 
> I haven't tested this, but I think adding `docker ps -a --filter "label=dbid=2"` to faultinject_python.sql will fix the running count problem.
> 
> ```
> \! ssh `psql -d ${PL_TESTDB} -c 'select address from gp_segment_configuration where dbid=2' -t -A` docker ps -a --filter "label=dbid=2" </dev/null | wc -l
> ```
> 
> PL/C will add many labels to the docker process.
> 
> https://github.com/greenplum-db/plcontainer/blob/5f9871867a88a928fc6fdbdfa63074e8899ea73a/src/plc_docker_api.c#L307-L311
> 
> the old test required the host did not have any docker container running. I think we should fix this for the developer experience.
> 
> ( I've never run a complete test locally.....

It worked thank you very much!

**Review by beeender — APPROVED** [review]

**Review by Sasasu — APPROVED** [review]

**Pull request merged**